docs/technical-doc/dummy-network-generation.rst

```diff
@@ -1,43 +1,85 @@
 ========================
-Dummy network generation
+Local network generation
 ========================
 
-How to launch a dummy network with custom settings and initial coins & rolls repartition
+How to launch a local network with custom settings and initial coins & rolls repartition
 ========================================================================================
 
-* get a private key and associated public key and address (see wallet_generate_private_key command), these will be referenced as PRI PUB and ADR
-* replace `massa-node/base_config/initial_rolls.json` content with
+**On your OS**
 
-.. code-block:: javascript
+Clone massa:
 
-    {
-        "ADR": 100
+.. code-block:: bash
+
+    git clone git@github.com:massalabs/massa.git
+
+Compile it with sandbox feature enabled:
+
+.. code-block:: bash
+
+    cd massa && cargo build --release --features sandbox
+
+Create a keypair in massa-client:
+
+.. code-block:: bash
+
+    cd massa-client && cargo run
+    wallet_generate_private_key
+
+For the rest of the tutorial we will use theses abreviations:
+
+- `PRIVK` : The private key you just generated
+- `PUBK` : The public key corresponding to PRIVK
+- `ADDR` : The address corresponding to PUBK
+
+Setup your node to use the private key you just generate as its private key and staking key:
+ * modify or create the file `massa-node/config/node_privkey.key` :
+
+ .. code-block:: bash
+
+    PRIVK
+
+ * modify or create the file `massa-node/config/staking_keys.json` :
+
+ .. code-block:: javascript
+
+    [
+        PRIVK
+    ]
+
+ * modify the file `massa-node/base_config/initial_ledger.json` :
+
+ .. code-block:: javascript
+
+    "ADDR": {
+        "balance": "250000000"
     }
 
-* replace `massa-node/base_config/initial_ledger.json` content with
+ * modify the file `massa-node/base_config/initial_rolls.json` :
 
-.. code-block:: javascript
+ .. code-block:: javascript
 
     {
-        "ADR": {
-            "balance": "250000000"
-        }
+        "ADDR": 100
     }
 
-* replace `massa-node/base_config/initial_sce_ledger.json` content with
+ * modify the file `massa-node/base_config/initial_sce_ledger.json` :
 
-.. code-block:: javascript
+ .. code-block:: javascript
 
-    {
-        "ADR": 250000000
+    "ADDR": {
+        "balance": "250000000"
     }
 
-* add or replace `massa-node/config/staking_keys.json`
+You can now launch your node :
 
-.. code-block:: javascript
+  .. code-block:: bash
+    
+    cd massa-node && cargo run --features sandbox
 
-    [
-    "PRI"
-    ]
+The network with your node all start in 10 seconds and you can now interact it with the CLI client like a testnet node.
+If you want to run multiple nodes on your local network you need to use the simulator that use docker all the documentation is here : https://github.com/massalabs/massa-network-simulator
+
+**On Docker**
```

Comment:
Why talk twice about this? You must be really proud of it 😄

Reply:
Done.